node/rpc/src/rpc_http_handler.rs

```diff
@@ -44,12 +45,11 @@ where
         ));
     }
 
-    let rpc_server = http_request.state();
-    let rpc_response = call_rpc_str(rpc_server.clone(), rpc_call).await?;
-    let http_response = HttpResponse::builder(200)
-        .body(rpc_response)
+    let result = call_rpc_str(rpc_server.clone(), rpc_call).await?;
+    let response = tide::Response::builder(200)
```

Comment:
Could qualify `tide::` imports here

Reply:
What do you mean?

Reply:
oh i meant to import the types at top.

Reply:
I prefer to leave it the way I changed it to so it's more explicit by not aliasing between http terms (request, response) and RPC terms (call, result).